components/user/UserTabs.js

```diff
@@ -81,73 +81,79 @@ export default function UserTabs({ tabs, setTabs, setUserData, userData }) {
   const getNested = (obj, args) => {
     return args.reduce((obj, level) => obj && obj[level], obj);
   };
+ 
+ return (
+  <div>
+   <div className="sm:hidden">
+    {tabs.length > 1 ? (
```

Comment:
I think this could be added above the `div` and therefore remove the need for the `<>` below

Reply:
Sorry @eddiejaoude for late response, had collage exams. I have made the required changes.